examples/views/button/paginator.py

```diff
@@ -16,39 +16,46 @@ def __init__(self, embeds: List[disnake.Embed]):
         # Current embed number.
         self.embed_count = 0
 
+        # Disables previous page button by default.
+        self.last_page.disabled = True
+
     @disnake.ui.button(label="Previous page", emoji="◀️", style=disnake.ButtonStyle.red)
-    async def next_page(self, button: disnake.ui.Button, interaction: disnake.MessageInteraction):
-        if self.embed_count == 0:  # If current embed is the first embed then, do not do anything.
-            pass
-        else:  # If current embed is not the first embed then, sends the preview embed.
-            self.embed_count -= 1
+    async def last_page(self, button: disnake.ui.Button, interaction: disnake.MessageInteraction):
```

Comment:
why is it suddenly the last page instead of the previous page?

Reply:
I don't want black doing this, but I can change it.
![image](https://user-images.githubusercontent.com/67214928/146432142-7e4b10f8-cfce-47dd-98f2-02f69798b976.png)

Reply:
`prev_page`